hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd

```diff
@@ -3312,6 +3312,16 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="nodeIdOffset" type="xs:long" default="0" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets the offset that will be added to the node ids assigned to the flake id generator instances.
+                    Default value is 0 and works fine for all green-field projects. A positive value might be set in
+                    some A/B deployment scenarios where there are 2 independent Hazelcast clusters generating ids with
+                    the same flake id instance.
```

Comment:
I still find this text confusing. The term "flake id generator instance" is confusing, I think of "instance" as of one distributed object, but we assign node ID to members. At the end you also use the term "instance", but you probably mean same-named object in two clusters. Also, we should write "ID" in upper case.

What about this?

>Sets the offset that will be added to the node ID assigned to cluster member for this generator. Might be useful in A/B deployment scenarios where you have cluster A which you want to upgrade. You create cluster B and for some time both will generate IDs and you want to have them unique. In this case, configure node ID offset for generators on cluster B.

Reply:
ok, i will update the text